<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSettingsDialog</class>
 <widget class="QWidget" name="FileSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>243</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3930</width>
    <height>2720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-color: #f0f0f0;
}
QFrame {
    background-color: #ffffff;
    border: 1px solid #cccccc;
}
QPushButton {
    background-color: #007acc;
    color: white;
    border: none;
    padding: 5px;
    border-radius: 3px;
}
QPushButton:hover {
    background-color: #005f9e;
}
QPushButton:pressed {
    background-color: #004a7c;
}
QTabWidget::pane {
    border: 1px solid #cccccc;
    background: #ffffff;
}
QTabBar::tab {
    background: #e0e0e0;
    border: 1px solid #cccccc;
    padding: 5px;
}
QTabBar::tab:selected {
    background: #ffffff;
    border-bottom: none;
}
   </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" rowspan="3" colspan="2">
    <widget class="QGroupBox" name="data_type_groupbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Symbols type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QRadioButton" name="int8_radio_button">
        <property name="text">
         <string>signed 8 complex</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">data_types_radio_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="int16_radio_button">
        <property name="text">
         <string>signed 16 complex</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">data_types_radio_group</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="float32_radio_button">
        <property name="text">
         <string>float 32 complex</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">data_types_radio_group</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>File name</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QLineEdit" name="choose_path_line_edit"/>
      </item>
      <item>
       <widget class="QToolButton" name="choose_path_tool_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="is_data_complex_checkbox">
     <property name="text">
      <string>complex data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3">
    <widget class="QGroupBox" name="signal_settings_groupbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Signal settings</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="carrier_labe">
        <property name="text">
         <string>Carrier</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="carrier_mhz_spinbox">
        <property name="suffix">
         <string> MHz </string>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="samplerate_label">
        <property name="text">
         <string>Samplerate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="samplerate_khz_spinbox">
        <property name="suffix">
         <string> KHz</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="ok_push_button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="data_types_radio_group"/>
 </buttongroups>
</ui>
